frontend/src/globals.css

```diff
@@ -98,3 +98,19 @@ li {
 a.svelte-1mtuslq.svelte-1mtuslq {
   display: none;
 }
+
+.App {
+  background: url('../public/desktop-background.png') no-repeat;
+  background-size: cover;
+  min-height: 100vh;
+  box-sizing: border-box;
+}
+
+@media (max-width: 550px) {
+  .withdrawall-wrapper {
```

Comment:
why this applied to withdrawall-wrapper? 
please fix and test next time before pushing, thanks